src/library/src/main/java/net/aichler/jupiter/internal/event/LoggingEventHandler.java

```diff
@@ -32,7 +32,7 @@
 /**
  * Writes dispatch events to a file for testing/debugging purposes.
  *
- * <h3>Example Usage</h3>
+ * <h2>Example Usage</h2>
```

Comment:
This change was necessary as the doc compiler on JDK 17 generated an error: 
heading used out of sequence: <H3>, compared to implicit preceding heading: <H1>